services/mailer/mail.go

```diff
@@ -222,7 +222,8 @@ func composeIssueCommentMessages(ctx *mailCommentContext, lang string, recipient
 	body, err := markdown.RenderString(&markup.RenderContext{
 		Ctx: ctx,
 		Links: markup.Links{
-			Base: ctx.Issue.Repo.HTMLURL(),
+			AbsolutePrefix: true,
```

Comment:
The mail, maybe `/routers/common/markup.go` and/or rendered webhook messages should be the only places with `AbsolutePrefix = true`. Some tests have `true` too but that should be changed in another PR.